modules/GUI.py

```diff
@@ -36,6 +36,34 @@ def __init__(self,screenX,screenY,marginX=None,marginY=None):
         self.keyTarget = None
         Base.GlobalKeyPython('#\nfrom GUI import *\nGUIRootSingleton.keyEvent()\n')
 
+        # VS Engine API version check ported verbatim from:
+        # https://github.com/vegastrike/Assets-Production/blob/master/modules/GUI.py 
+        hasVersion = hasattr(VS, 'EngineVersion')
+        # get the engine version tuple in a displayable format
+        # if it needs to be compared, then use the original tuple version
+        ev = (
+            VS.EngineVersion().GetVersion()
+            if hasVersion
+            else (0, 7, 0, 'unknown') # 0.7.x was the last version without this API
+        )
+        engineVersion = '.'.join(
+            [
+                str(i)
+                for i in ev
+            ]
+        )
+
+        apiVersion = (
+            VS.EngineVersion().GetAssetAPIVersion()
+            if hasVersion
+            else 0
+        )
+
+        trace(TRACE_WARNING, "::: What's in VS object %s :::" %(dir(VS)))
+        trace(TRACE_WARNING, "::: Engine Version {0} :::".format(engineVersion))
+        trace(TRACE_WARNING, "::: Asset API Version {0} :::".format(apiVersion))
+        # end VS Engine API version check
```

Comment:
@stephengtuggy we (Vegastrike team) should post this code on its own and recommend people copy it in from that source so that we don't have to remember it for each game asset set.

Reply:
This is the note